ipaserver/install/server/replicainstall.py

```diff
@@ -1555,7 +1566,7 @@ def install(installer):
     # Enable configured services and update DNS SRV records
     service.enable_services(config.host_name)
     api.Command.dns_update_system_records()
-    ca_servers = service.find_providing_servers('CA', api.Backend.ldap2, api)
+    ca_servers = find_providing_servers('CA', api.Backend.ldap2, api)
```

Comment:
Does it make more sense to make find_providing_servers() public or use the server_role mechanism to see if there is more than one CA installed?